.github/workflows/test.yml

```diff
@@ -23,16 +23,23 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-
+      - name: Get Tools Versions
+        id: tools-versions
+        run: |
+          current_helm_version="$(curl --silent --location https://raw.githubusercontent.com/jenkins-infra/docker-helmfile/main/Dockerfile  | grep 'ARG' | grep 'HELM_VERSION=' | sort -u | cut -d'=' -f2)"
+          echo "CURRENT_HELM_VERSION=${current_helm_version}" >> $GITHUB_OUTPUT
       - name: Set up Helm
         uses: azure/setup-helm@v3
+        with:
+          version: "${{ steps.tools-versions.outputs.CURRENT_HELM_VERSION }}"
 
       - name: install helm unittests
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           helm env
-          helm plugin install https://github.com/quintush/helm-unittest
+          # Repeat 3 times to fight against network errors in  GHA runners
```

Comment:
🎉  :trollface:

Reply:
We are here to do nasty hotfixes and to chew bubble gum. And we outta bubble gum